### ISSUE TITLE
Is there any solution to regenerate a JSTS based on the latest version of JTS?

### ISSUE THREAD

**Issue by spskauynnda** [bjornharrtell/jsts#527]
Thanks for the awesome library.

JTS has a major upgrade to 1.20.0, which can solve many geo operation bugs I encountered. Is there any way to regenerate a JSTS based on the latest version of JTS?

**Comment by bjornharrtell** [comment]
Yes but non trivial, see https://github.com/bjornharrtell/jsts/issues/488.
